src/packages/cli/src/__tests__/__snapshots__/dotenv-1-custom-schema-path.test.ts.snap

```diff
@@ -4,7 +4,7 @@ exports[`should read .env file in root folder and custom-path 1`] = `
 Array [
   Array [
     Environment variables loaded from /tmp/dir/.env
```

Comment:
General question: Why is this an array of arrays?

Reply:
Because I reused logic from other tests

Reply:
Fixed :)